cluster/olm/ceph/assemble/metadata-common.yaml

```diff
@@ -89,7 +89,7 @@ metadata:
           },
           "spec": {
             "cephVersion": {
-              "image": "ceph/ceph:v14.2.1-20190430"
+              "image": "ceph/ceph:latest-nautlius-devel"
```

Comment:
the repo name and the tag name are incorrect.

It's ceph/daemon-base:latest-nautilus-devel